src/common/helpers/data-extract/data-extract.js

```diff
@@ -99,38 +99,40 @@ export const getTbLicenceType = (application) => {
   const destinationType = getDestinationType(application)
 
   if (
-    originType?.value !== 'afu' &&
-    !isTbRestricted(originType) &&
-    isTbRestricted(destinationType)
+    (originType?.value === 'market' && isTbRestricted(destinationType)) ||
+    (originType?.value === 'unrestricted-farm' &&
+      isTbRestricted(destinationType)) ||
+    (originType?.value === 'after-import-location' &&
+      isTbRestricted(destinationType))
   ) {
     return 'TB15'
   }
 
-  if (isTbRestricted(originType) && isTbRestricted(destinationType)) {
-    return 'TB16'
-  }
-
-  if (isTbRestricted(originType) && destinationType?.value === 'afu') {
-    return 'TB16e'
-  }
-
   if (
     isTbRestricted(originType) &&
-    destinationType?.value === 'dedicated-sale'
+    isTbRestricted(destinationType) &&
+    destinationType?.value !== 'iso-unit'
```

Comment:
`isTbRestricted(destinationType)` implies that `destinationType?.value !== 'iso-unit'`, doesn't it?